[2024-10-31T20:54:42.893+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-31T20:54:42.922+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.first_task scheduled__2024-10-30T00:00:00+00:00 [queued]>
[2024-10-31T20:54:42.951+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.first_task scheduled__2024-10-30T00:00:00+00:00 [queued]>
[2024-10-31T20:54:42.960+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-31T20:54:42.995+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): first_task> on 2024-10-30 00:00:00+00:00
[2024-10-31T20:54:43.004+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'first_task', 'scheduled__2024-10-30T00:00:00+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpui4_55kk']
[2024-10-31T20:54:43.012+0000] {standard_task_runner.py:105} INFO - Job 101: Subtask first_task
[2024-10-31T20:54:43.013+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=191) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-31T20:54:43.015+0000] {standard_task_runner.py:72} INFO - Started process 192 to run task
[2024-10-31T20:54:43.093+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v3.first_task scheduled__2024-10-30T00:00:00+00:00 [running]> on host 2d730dc1e71d
[2024-10-31T20:54:43.337+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='johnwell' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-30T00:00:00+00:00'
[2024-10-31T20:54:43.338+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-31T20:54:43.407+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-31T20:54:43.409+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-10-31T20:54:43.440+0000] {subprocess.py:86} INFO - Output:
[2024-10-31T20:54:43.455+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-10-31T20:54:43.462+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-31T20:54:43.534+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-31T20:54:43.535+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=first_task, run_id=scheduled__2024-10-30T00:00:00+00:00, execution_date=20241030T000000, start_date=20241031T205442, end_date=20241031T205443
[2024-10-31T20:54:43.597+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-31T20:54:43.627+0000] {taskinstance.py:3900} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-10-31T20:54:43.638+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
